docs/sdkx-react-native/outbound-support.mdx

```diff
@@ -0,0 +1,104 @@
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Outbound Support {#outbound-support}
+
+<Intro>
+
+With outbound support, you can proactively engage with consumers to solve problems within the app. Read more about the feature **[here](https://support.helpshift.com/hc/faq/1163-outbound-support-create-outbound-support-link)**.
+
+</Intro>
+
+<Admonition type="info" title="Note">
+
+All the public APIs in the SDK should be called after initializing the SDK via [Install API](/sdkx-react-native/getting-started#initializing)
+
+</Admonition>
+
+The steps to use this feature are the following -
+
+To generate the link for outbound support, on your Helpshift dashboard, go to **Settings** > **Workflows** > **Outbound Support**.
+
+![](/static/books/helpshiftx/outbound-support.png)
+
+You should see a **Create link** button. Click on the Create link button and select an action like **Chat, Help Center, Single FAQ or FAQ Section** and other data like **CIFs, Tags, First User Message** you want to send as payload to Helpshift SDK.
+
+<Image src="/static/books/helpshiftx/generate-link-box.png" width="full" />{" "}
+
+At last, you will get a URL-encoded payload link. Send this link to your end-users embedded in a notification payload using your existing Push notification system.
+
+YOUR_APP_IDENTIFIER: This can be any unique string that identifies your app. For example, like the scheme, you would use deep link URLs for your app like myApp, myAppSupport, etc.
+
+<Image src="/static/books/helpshiftx/proactive-link.png" width="full" />{" "}
+
+## Delegate push notification data to Helpshift {#delegate-notification-data}
+
+To pass the outbound support data to Helpshift, follow these steps -
+
+1. Send push notification to the users you want to give outbound support using your app's existing push notification system
+
+2. In your app, handle this notification such that when a user opens the app through notification, you pass the outbound support link in your notification data to Helpshift SDK by calling `handleProactiveLink(String link,{})`.
+
+3. We will read the data from the link you provided and open Helpshift support with the configurations you provided from outbound support dashboard.
+
+For example -
+
+_Following code shows how to handle incoming push notification and posting it on the notification bar of the device_
+
+```javascript
+import {
+  handleProactiveLink,
+  handlePush
+} from 'helpshift-plugin-sdkx-react-native';
+
+const notificationData = remoteMessage.data;
+if (notificationData != undefined) {
+      const isHandleProactiveLink = notificationData != undefined && Object.hasOwn(notificationData, 'helpshift_proactive_link');
+      const doesNotificationDataHaveOrigin =
+        notificationData.hasOwnProperty('origin') &&
+        notificationData?.origin == 'helpshift';
+      if (isHandleProactiveLink) {
+        handleProactiveLink(notificationData.helpshift_proactive_link || '', <LOCAL_API_CONFIG>);
```

Comment:
notificationData.helpshift_proactive_link || ''
we cannot have an empty string in link, so dont even mention it here

Reply:
Deleted.